### PR TITLE
Add Whats New content for Platform 5.4

### PR DESCRIPTION
**Pull request by Serdaro**

### CODE REVIEW

File: docs/modules/ROOT/pages/whats-new.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+**JDBC SQL Connector (SQL over JDBC)** is now generally available. In addition to MySQL and PostgreSQL, JDBC SQL connector now also supports Oracle and Microsoft SQL Server databases. See xref:sql:mapping-to-jdbc.adoc#data-type-mapping-between-hazelcast-and-mssql[JDBC Connector] to learn how data types in these two databases are mapped to Hazelcast.
+
+== Release Notes
+
+For detailed release notes that include new features and enhancements, breaking changes, deprecations and other fixes, see xref:release-notes:5-4-0.adoc[].
+
+== Hazelcast Command Line Client (CLC)
+
+You can use the Hazelcast Command Line Client (CLC) to connect to and interact with clusters on Hazelcast Platform and Hazelcast Cloud directly from the command line or through scripts.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/whats-new.adoc:49]:
```suggestion
You can use the Hazelcast Command Line Client (CLC) to connect to, and interact with, clusters on Hazelcast Platform and Hazelcast Cloud directly from the command line or through scripts.
```

File: docs/modules/ROOT/pages/whats-new.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+You can now import a subset of the Java client config configuration. You can also now define custom template actions using advanced scripts, and add a success note when a project is created successfully.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/whats-new.adoc:51]:
```suggestion
You can now import a subset of the Java client configuration. You can also now define custom template actions using advanced scripts, and add a success note when a project is created successfully.
```

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/whats-new.adoc [hunk 3]
```diff
@@ -4,50 +4,58 @@
...
+**Mongo SQL Connector (SQL over MongoDB)** is now generally available. With the addition of security support, you can now secure your data connection with TLS when creating MongoDB mappings over SQL. For more information, see xref:sql:mapping-to-mongo.adoc[].
 
-**JDBC SQL Connector (SQL over JDBC)** is now generally available. In addition to MySQL and PostgreSQL, JDBC SQL connector now also supports Oracle and Microsoft SQL Server databases. See xref:sql:mapping-to-jdbc.adoc#data-type-mapping-between-hazelcast-and-mssql[JDBC Connector] to learn how data types in these two databases are mapped to Hazelcast.
+**JDBC SQL Connector (SQL over JDBC)** is now generally available. In addition to MySQL and PostgreSQL, the JDBC SQL connector now also supports Oracle and Microsoft SQL Server databases. For more ifnormation, see xref:sql:mapping-to-jdbc.adoc#data-type-mapping-between-hazelcast-and-mssql[Mapping between Hazelcast and MSSQL].
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/whats-new.adoc:47]:
```suggestion
**JDBC SQL Connector (SQL over JDBC)** is now generally available. In addition to MySQL and PostgreSQL, the JDBC SQL connector now also supports Oracle and Microsoft SQL Server databases. For more information, see xref:sql:mapping-to-jdbc.adoc#data-type-mapping-between-hazelcast-and-mssql[Mapping between Hazelcast and MSSQL].
```

File: docs/modules/ROOT/pages/whats-new.adoc [hunk 4]
```diff
@@ -4,50 +4,58 @@
...
 
 For more information on how to enable and configure TPC, see xref:cluster-performance:thread-per-core-tpc.adoc[].
 
-== Compute Isolation with User Code Namespaces [.enterprise]*Enterprise* 
+== Manage resources effectively with User Code Namespaces
+[.enterprise]*Enterprise* 
 
-User Code Namespaces provide a container for Java classpath resources such as user code, and accompanying artifacts such as property files. This allows you to effectively manage access to resources in different namespaces through either static or dynamic configuration. This is an Enterprise feature that extends and replaces the now deprecated xref:clusters:legacy-ucd.adoc[User Code Deployment] functionality.
+User Code Namespaces provide a container for Java classpath resources, such as user code and accompanying artifacts such as property files. This enables you to effectively manage access to resources in different namespaces through either static or dynamic configuration. This is an Enterprise feature that extends and replaces the now deprecated xref:clusters:legacy-ucd.adoc[User Code Deployment] functionality.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/whats-new.adoc:37]: Just saying:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/enable-enables
https://developers.google.com/style/word-list#enable
https://www.plainenglish.co.uk/the-a-z-of-alternative-words.html#E

I note that Google and MS style guides do not like allows either, so perhaps "this means that you can effectively...."

;-)

I feel this is especially important on this page where enables is used both to mean allows and to mean turn on

**Review by rebekah-lawrence — COMMENTED** [review]
continuing the enables disagreement, and a typo

File: docs/modules/ROOT/pages/whats-new.adoc (hunk 4 above)
Comment (oliverhowell) [reply docs/modules/ROOT/pages/whats-new.adoc:37]: lol ok, maybe there's at third way out this allows/enables hell

File: docs/modules/ROOT/pages/whats-new.adoc [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+== Hazelcast Command Line Client (CLC)
+
+You can use the Hazelcast Command Line Client (CLC) to connect to and interact with clusters on Hazelcast Platform and Hazelcast Cloud directly from the command line or through scripts.
+
+With Hazelcast CLC 5.3.7 you can now import a subset of the Java client configuration. You can also define custom template actions using advanced scripts, and add a success note when a project is created successfully.
+
+For detailed release notes that include new features and fixes, see xref:clc:release-notes-5.3.7.adoc[Hazelcast CLC 5.3.7].
+
+To get started with Hazelcast CLC, see xref:clc:install-clc.adoc[].
```
Comment (oliverhowell) [inline docs/modules/ROOT/pages/whats-new.adoc:61]: Another link not resolving in preview: xref:clc:install-clc.adoc[]

**Review by oliverhowell — APPROVED** [review]

**Comment by Serdaro** [comment]
Closing this one. Will create the PR for 5.4 branch.

**Pull request closed without merging**